bc_obps/registration/schema/v2/operation.py

```diff
@@ -77,9 +77,10 @@ class Meta:
 
 
 class OperationInformationIn(ModelSchema):
+    name: str
     registration_purpose: Optional[Operation.Purposes] = None
     regulated_products: Optional[List[int]] = None
-    activities: List[int]
+    activities: Optional[List[int]] = None
```

Comment:
This is the only field that isn't already optional, so perhaps it's not worth making a separate EIO schema